File: cukcuk clone/app/src/main/java/vn/com/misa/cukcukstarterclone/ui/login/register/RegisterContract.java
package vn.com.misa.cukcukstarterclone.ui.login.register;

import vn.com.misa.cukcukstarterclone.base.BaseContract;

/**
 * @created_by KhanhNQ on 08-Feb-2021.
 */
public class RegisterContract {
    interface View extends BaseContract.View {
        void registerSuccess();
    }

    interface Presenter extends BaseContract.Presenter<View> {
        void register(String username, String password);
    }
}

File: app/src/main/java/com/misa/fresher/ui/login/register/RegisterContract.kt
package com.misa.fresher.ui.login.register

import com.misa.fresher.base.BaseContract
import com.misa.fresher.data.model.User

class RegisterContract :BaseContract {
    interface View : BaseContract.View{
        fun registerSuccess()
    }
    interface Presenter: BaseContract.Presenter<View>{
        fun register(user: User)
    }
}